### PR TITLE
FIX: fix save fiducials in mne coreg with scaled MRI (#5491)

### LINKED ISSUE #5491: mne coreg save with scaled mri fails if fiducials haven't been saved
Hi, when trying to save a -trans file from MNE coreg after scaling the MRI the dialog box doesn't open if the fiducials file has not been saved.

**Problem:**
I tried tracking the problem and I think it arises because the dialog box that asks if one wants to save the fiducials fails to open due to:

 _coreg_gui.py; line 926:_
"rc = confirm(None, msg, title, cancel=True, default=CANCEL,
                    parent=self.info.ui.control)"

because the function "confirm" takes _parent_ as first argument and is also receiving it as keyword argument:

_/pyface/confirmation_dialog.py; line 22:_
"def confirm(parent, message, title=None, cancel=False, default=NO):" 

**Possible solutions:**
I've tried fixing it and I'v seen that both of the following changes fix the issue but I don't  know which one is preferable:

"rc = confirm(None, msg, title, cancel=True, default=CANCEL)"
                  
"rc = confirm(self.info.ui.control, msg, title, cancel=True, default=CANCEL)"

I'll be happy to make a pull request if you think this is ok and if you tell me which way is preferable.

Thanks for everything, cheers,

Eze

**Error Traceback:**
Exception occurred in traits notification handler for object: <mne.gui._coreg_gui.CoregPanel object at 0x7fc2ab01dca8>, trait: save, old value: <undefined>, new value: 0
Traceback (most recent call last):
  File "/home/eze/soft/anaconda3/envs/mne3/lib/python3.6/site-packages/traits/trait_notifiers.py", line 519, in _dispatch_change_event
    self.dispatch( handler, *args )
  File "/home/eze/soft/anaconda3/envs/mne3/lib/python3.6/site-packages/traits/trait_notifiers.py", line 614, in dispatch
    handler( *args )
  File "/home/eze/soft/anaconda3/envs/mne3/lib/python3.6/site-packages/traitsui/ui.py", line 1023, in dispatch
    self.method(self.info)
  File "/home/eze/soft/anaconda3/envs/mne3/lib/python3.6/site-packages/mne/gui/_coreg_gui.py", line 927, in object_save_changed
    parent=self.info.ui.control)
TypeError: confirm() got multiple values for argument 'parent'

**Sys info:**
Platform:      Linux-4.15.0-33-generic-x86_64-with-debian-buster-sid
Python:        3.6.6 |Anaconda, Inc.| (default, Jun 28 2018, 17:14:51)  [GCC 7.2.0]
Executable:    /home/eze/soft/anaconda3/envs/mne3/bin/python
CPU:           x86_64: 8 cores
Memory:        15.6 GB
mne:           0.17.dev0
numpy:         1.15.0 {blas=mkl_rt, lapack=mkl_rt}
scipy:         1.1.0
matplotlib:    2.2.2 {backend=Qt5Agg}
sklearn:       0.19.1
nibabel:       2.3.0
mayavi:        4.6.1 {qt_api=pyqt5}
cupy:          Not found
pandas:        0.23.3
dipy:          Not found

### PR DESCRIPTION
**Pull request by ezemikulan**
#### Reference issue
Fixes #5491 

#### What does this implement/fix?
The dialog box asking to save the fiducials file didn't open when trying to save a -trans file with a scaled MRI, which caused the saving procedure to fail. This fixes the problem.

### CODE REVIEW

**Comment by agramfort** [comment]
I let @larsoner judge if it's correct or not :)

File: doc/whats_new.rst [hunk 1]
```diff
@@ -110,6 +110,8 @@ Bug
 
 - Fix event sample number increase when combining many Epochs objects with :func:`mne.concatenate_epochs` with  by `Jasper van den Bosch`_
 
+- Fix error in mne coreg when saving with scaled MRI if fiducials haven't been saved by `Ezequiel Mikulan`_
```
Comment (larsoner) [inline doc/whats_new.rst]: Almost the right pattern matching here :)

You need to add an entry at the bottom so that there is a URL

**Comment by ezemikulan** [comment]
Thanks for correcting my begginer's mistakes :)

 I updated the code and I see now that Travis showed an error:
_"No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
(...)
The build has been terminated"_

I suppose I should run the test again but as far as I've read it's only possible for users with write access. Can you do it? Or should I close the PR and open a new one?

**Comment by larsoner** [comment]
Restarted

**Comment by larsoner** [comment]
Thanks @ezemikulan

**Pull request merged**